destinations/data/quickwit.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: internal.odigos.io/v1beta1
+kind: Destination
+metadata:
+  type: quickwit
+  displayName: Quickwit
+  category: self hosted
+spec:
+  image: quickwit.svg
+  signals:
+    traces:
+      supported: true
+    metrics:
+      supported: false
```

Comment:
the go config file will attempt to export metrics to this destination but it is not supported in the yaml. should this be `true`?

Reply:
I think Quickwit is not supporting metrics for now (it should change in the future), cc @fmassot if you can confirm.

Reply:
I removed the metrics part from the go config file